typedoc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://typedoc.org/schema.json",
+    "readme": "README.md",
+    "includeVersion": true,
+    "entryPointStrategy": "expand",
+    "entryPoints": ["src/AEG"], //This needs to be changed once more classes are added outside AEG
```

Comment:
I'd suggest pinning this in discord. Or make an important pull request checklist channel and put it there. Unless you don't foresee that many classes being added in the future.